src/scaffoldmaker/utils/tubenetworkmesh.py

```diff
@@ -1050,6 +1261,137 @@ def setRimElementId(self, e1, e2, e3, elementIdentifier):
             self._rimElementIds[e2] = [[None] * self._elementsCountAround for _ in range(elementsCountRim)]
         self._rimElementIds[e2][e3][e1] = elementIdentifier
 
+    def getBoxElementId(self, e1, e2, e3):
+        """
+        Get a box element ID.
+        :param e1: Element index across core box major / d2 direction.
+        :param e2: Element index along segment.
+        :param e3: Element index across core box minor / d3 direction.
+        :return: Element identifier.
+        """
+        return self._boxElementIds[e2][e3][e1]
+
+    def setBoxElementId(self, e1, e2, e3, elementIdentifier):
+        """
+        Set a box element ID. Only called by adjacent junctions.
+        :param e1: Element index across core box major / d2 direction.
+        :param e2: Element index along segment.
+        :param e3: Element index across core box minor / d3 direction.
+        :param elementIdentifier: Element identifier.
+        """
+        self._elementsCountCoreBoxMajor
+        self._elementsCountCoreBoxMinor
+        if not self._boxElementIds[e2]:
+            elementsCountRim = self.getElementsCountRim()
+            self._boxElementIds[e2] = [
+                [None] * self._elementsCountCoreBoxMinor for _ in range(self._elementsCountCoreBoxMajor)]
```

Comment:
Line 1285 elementsCountRim is also not being used so we can remove it.

Reply:
Removed.